docs/Crypto.php

```diff
@@ -1,8 +1,11 @@
 <?php
+
+namespace Crypto{
```

Comment:
Couldn't this just be `namespace Crypto;` or is it not working with IDE parser...?

Reply:
just curios - not really an issue though... :) if it doesn't work, then it's fine - possibly there should be a space before `{` and the content indented but not really an issue either...

Reply:
You're right, I'll fix that.